src/main/scala/com/atomist/rug/functions/travis/TravisFunction.scala

```diff
@@ -11,3 +11,9 @@ trait TravisFunction extends Rug {
   lazy val travisEndpoints: TravisEndpoints = new RealTravisEndpoints
 
 }
+
+object TravisFunction {
+
+  final val githubTokenPath = "github://user_token?scopes=repo,read:org,user:email"
```

Comment:
I'm not sure how all this works in the bot, but if you haven't already, it might be worth checking with @tanya-coding if this is correct.